addons/assigngear/functions/fn_replaceSecondaryWeapon.sqf

```diff
@@ -14,7 +14,7 @@
  * Replaces a units' secondary weapon
  */
  #include "\x\tmf\addons\assignGear\script_component.hpp"
- params ["_unit","_weaponArray","_items"];
+ params ["_unit","_weaponArray","_items","_magazineArray"];
```

Comment:
Probably better to edit params to be
` params ["_unit","_weaponArray","_items",["_magazineArray",[]]];`

Infact, that goes for all of the parameters there